sqrl-planner/sqrl-common/src/main/java/com/datasqrl/calcite/type/Vector.java

```diff
@@ -0,0 +1,21 @@
+package com.datasqrl.calcite.type;
+
+import org.apache.calcite.sql.type.SqlTypeName;
+
+public class Vector extends BridgingFlinkType {
```

Comment:
Can we call this VectorType for clarity and rename FlinkVectorType to FlinkVector since it represents the actual vector object?